bloom/bloomfilter.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package bloom
+
+import (
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/pkg/errors"
+)
+
+type (
+	// BloomFilter interface
+	BloomFilter interface {
+		// Add 32-byte key into bloom filter
+		Add(hash.Hash256)
+		// Exist checks if a key is in bloom filter
+		Exist(hash.Hash256) bool
+		// Bytes returns the bytes of bloom filter
+		Bytes() []byte
+		// SetBytes copies the bytes into bloom filter
+		SetBytes([]byte) error
```

Comment:
We may not need SetBytes api given BloomFilterFromBytes exists

Reply:
fixed in new push